bears/vcs/git/GitCommitBear.py

```diff
@@ -291,6 +291,9 @@ def check_issue_reference(self, body,
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
+        if not re.search(self.SUPPORTED_HOST_KEYWORD_REGEX[host], body):
+            return
```

Comment:
It won't match, deliberately reporting the result that body doesn't contain any issue reference.

```
README: Modify installation procedure.

Add the details about setting up ``git``
```
The above commit, will be reported false positive for errors, which is unexpected, right?